Lombiq.Tests.UI.Samples/Tests/BasicVisualTests.cs

```diff
@@ -0,0 +1,173 @@
+using Lombiq.HelpfulLibraries.Common.Utilities;
+using Lombiq.Tests.UI.Attributes;
+using Lombiq.Tests.UI.Constants;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+using Shouldly;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Processing;
+using System.Globalization;
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Lombiq.Tests.UI.Samples.Tests;
+
+// In this basic test we will check visually the rendered content.
+public class BasicVisualTests : UITestBase
+{
+    public BasicVisualTests(ITestOutputHelper testOutputHelper)
+        : base(testOutputHelper)
+    {
+    }
+
+    // Checking that everything is OK with the branding on navbar of homepage.
+    // For this magic we are using ImageSharp.Compare package. You can find more info about it here:
```

Comment:
Please commit these suggestions, if you agree with them, directly from GH, so they're auto-resolved. Otherwise I'm wasting time on checking them one by one.